presto-native-execution/presto_cpp/presto_protocol/WriteProtocol.cpp

```diff
@@ -35,25 +35,27 @@ std::string makePartitionUpdate(
       hiveWriterParameters,
       "Presto commit protocol only supports Hive commits at the moment.")
   // clang-format off
-  auto partitionUpdateJson = folly::toJson(
-   folly::dynamic::object
-     ("name", "")(
-      "updateMode",
-      HiveWriterParameters::updateModeToString(
-          hiveWriterParameters->updateMode()))("writePath", hiveWriterParameters->writeDirectory())
-     ("targetPath", hiveWriterParameters->targetDirectory())
-     ("fileWriteInfos", folly::dynamic::array(
-       folly::dynamic::object
-         ("writeFileName", hiveWriterParameters->writeFileName())
-         ("targetFileName", hiveWriterParameters->targetFileName())
-         ("fileSize", 0)))
-     ("rowCount", numWrittenRows)
-     // TODO(gaoge): track and send the fields when inMemoryDataSizeInBytes, onDiskDataSizeInBytes
-     // and containsNumberedFileNames are needed at coordinator when file_renaming_enabled are turned on,
-     // https://fburl.com/18vim7k4.
-     ("inMemoryDataSizeInBytes", 0)
-     ("onDiskDataSizeInBytes", 0)
-     ("containsNumberedFileNames", true));
+ auto partitionUpdateJson = folly::toJson(
+     folly::dynamic::object
+         ("name", hiveWriterParameters->partitionName().value_or(""))
```

Comment:
Doing some formatting, but only the line of the "name" field changes, for this function